tasmota/include/tasmota_template_legacy.h

```diff
@@ -471,7 +471,10 @@ const uint16_t kGpioConvert[] PROGMEM = {
   AGPIO(GPIO_IEM3000_TX),     // IEM3000 Serial interface
   AGPIO(GPIO_IEM3000_RX),     // IEM3000 Serial interface
   AGPIO(GPIO_ZIGBEE_RST),     // Zigbee reset
-  AGPIO(GPIO_DYP_RX)
+  AGPIO(GPIO_DYP_RX),
```

Comment:
Read the header:

// ATTENTION: No additions are supported

So remove your changes to this file.